explorer/nginx/conf.d/nginx.conf.envsubst

```diff
@@ -70,6 +70,19 @@ http {
             add_header Cache-Control "public, max-age=1";
             include conf.d/enable-cors.conf;
         }
+
+         # expose current universe for resources to fetch
```

Comment:
What this change is for? I don't really get it

Reply:
This is regarding the architecture.
Unfortunatly, by filling in the `CORE_UNIVERSE` inside the `core` cloud-init file, it was impossible to transfer it over the AWS API.

So, architecture was changed to make the explorer provide the universe, and the core nodes grab the universe from an endpoint.